app/views/layouts/_flash.html.erb

```diff
@@ -1,14 +1,14 @@
 <% if flash.present? %>
-    <div class="row-fluid alerts">
-      <% flash.each do |name, message| %>
-          <div class="alert fade in <%= flash_alert_class(name) %>">
-            <div class="container">
-              <button class="close" data-dismiss="alert" aria-label="Close">
-                <span aria-hidden="true">&times;</span>
-              </button>
-              <%= message %>
-            </div>
-          </div>
-      <% end %>
-    </div>
+  <div class="row-fluid alerts">
+    <% flash.each do |name, message| %>
+      <div class="alert <%= flash_alert_class(name) %> mb-0">
```

Comment:
`alert` class has default margin. So I add `mb-0` to avoid empty margin between navbar and alert. 
![screen shot 2561-06-07 at 11 25 23](https://user-images.githubusercontent.com/3861498/41078435-05f7c484-6a47-11e8-93d6-9e3b67848b45.png)

Reply:
generally alerts go below the navbars, is the template set up for the alerts to be above by default?

Reply:
@hiattp 
I'm usually see navbar placed at the top of screen what means it will be above the navbar. But in your case it totally makes sense to keep that margin. I'll redo this change.

Reply:
So @vvmarulin this will not make the alert dismissible, right? Maybe it's just me but I like to close them if possible, must be OCD.

Reply:
It seems like alerts even in the template should be dismissable. this is expected bootstrap behavior since virtually all bootstrap-implemented alerts are dismissable, so while it could be construed as scope creep in the template, since the template supports bootstrap in general it seems like making these dismissable isn't over-reach

Reply:
@janklimo 
Fixed. Added ' alert-dismissible'. Remove `mb-0` that I added earlier.